src/Pate/Discovery.hs

```diff
@@ -269,8 +256,9 @@ associateFrames ::
   forall sym arch v.
   SimBundle sym arch v ->
   MCS.MacawBlockEndCase ->
-  EquivM sym arch (PSS.AssumptionSet sym v)
-associateFrames bundle exitCase = PPa.catBins $ \get -> do
+  Bool ->
```

Comment:
At the very least we should document what this boolean is. Better, we can make it into an ADT that is descriptive at the call site